src/components/VtkThreeView.vue

```diff
@@ -42,9 +42,9 @@ export default {
 
     const {
       sceneSources,
-      worldOrientation,
+      imageConfig,
```

Comment:
How about `imageParams`? `imageConfig` makes me thing of a software-level configuration for images, not image metadata.

I realize this is a big search and replace but may make things clearer overall.

Reply:
+1 I agree that `imageParams` is clearer than `imageConfig`.